src/sagas.js

```diff
@@ -426,6 +426,13 @@ function* changeVmIcon (action) {
 
```

Comment:
Can we just replace the rest of this method by calling `fetchSingleVm()`?

Reply:
Right, whole `if` is not needed, just initiate refresh of the single VM